packages/core/src/utils/Error.ts

```diff
@@ -433,7 +442,8 @@ canvas {
 };
 
 const showCycles = (cycles: string[][]) => {
-  const pathString = (path: string[]) => path.join(" -> ");
+  // repeats the cycle start again
+  const pathString = (path: string[]) => [...path, path[0]].join(" <-> ");
```

Comment:
Could you explain the reasoning behind changing this from a single arrow to a double arrow? The former feels more accurate to me.

Reply:
mostly trying to see how it'd look on the screen. the reasoning is that you can go in any direction because it's a cycle

Reply:
Ah gotcha; that's not true though, like if you have the cycle `1 -> 2 -> 3 -> 1`, that's different from the cycle `1 -> 3 -> 2 -> 1`. Is it OK if we change it back?